package.json

```diff
@@ -2,34 +2,64 @@
 	"name": "lisk-js",
 	"version": "0.4.5",
 	"description": "JavaScript library for sending Lisk transactions from the client or server",
-	"main": "./dist-node/index.js",
+	"author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+	"license": "GPL-3.0",
+	"keywords": [
+		"lisk",
+		"blockchain"
+	],
+	"homepage": "https://github.com/LiskHQ/lisk-js#readme",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/LiskHQ/lisk-js.git"
+	},
+	"bugs": {
+		"url": "https://github.com/LiskHQ/lisk-js/issues"
+	},
+	"engines": {
+		"node": "=6.12.3",
+		"npm": "=3.10.10"
+	},
+	"main": "dist-node/index.js",
 	"scripts": {
 		"prestart": "./bin/prestart.sh",
 		"start": "./bin/start.sh",
-		"test": "nyc mocha test --recursive",
-		"test:watch": "chokidar \"./(src|test)/**/*.js\" -c \"mocha test --recursive\" --initial",
+		"clean": "rm -rf ./node_modules ./dist-browser/lisk-js.js ./dist-browser/lisk-js.min.js ./dist-node",
+		"clean:browsertest": "rm -r browsertest/src browsertest/test",
+		"format": "prettier --write \"browsertest/{setup,runTests}.js\" \"{cypress/integration,src,test}/**/*.js\"",
+		"lint": "eslint .",
+		"lint:fix": "npm run lint -- --fix",
+		"test": "NODE_ENV=test nyc mocha test",
+		"test:watch": "npm test -- --watch",
+		"test:watch:min": "npm run test:watch -- --reporter=min",
 		"test:browser": "cypress run --env ROOT_DIR=\"${PWD##*/}\"",
-		"cover": "nyc report --reporter=lcov",
-		"eslint": "grunt eslint",
-		"prettier": "prettier --write \"browsertest/{setup,runTests}.js\" \"{cypress/integration,src,test}/**/*.js\"",
+		"cover": "if [ -z $JENKINS_HOME ]; then npm run cover:local; else npm run cover:ci; fi",
+		"cover:base": "NODE_ENV=test nyc report",
+		"cover:local": "npm run cover:base -- --reporter=html --reporter=text",
+		"cover:ci": "npm run cover:base -- --reporter=text-lcov | coveralls -v",
 		"serve:browsertest": "http-server browsertest",
-		"jenkins": "grunt jenkins --verbose",
-		"build": "grunt build",
+		"prebuild:node": "rm -r dist-node/* || mkdir dist-node | echo",
```

Comment:
Should there also be a `prebuild:browser` script? I think it got lost from the `Gruntfile.js`.